src/bp/migrations/v12_21_0-1617829329-mapping_migration.ts

```diff
@@ -0,0 +1,28 @@
+import * as sdk from 'botpress/sdk'
```

Comment:
Two questions:

 - Are you sure about the version of this migration?
 - In my opinion, this migration should only alter the `local` column and change its type to `uuid` since this table might already exists. What do you think?

Reply:
For the version I checked with @EFF so if we don't bump the version on the next release we'll change it to 12.20.2.